doc/dev/cephfs-mirroring.rst

```diff
@@ -17,7 +17,7 @@ Key Idea
 --------
 
 For a given snapshot pair in a directory, `cephfs-mirror` daemon will rely on
-`CephFS Snapdiff Feature` to identify changes in a directory tree. The diffs
+`CephFS Snapdiff Feature`_ to identify changes in a directory tree. The diffs
```

Comment:
Prepend "the" please.

Also, is this a location where we can use :ref: instead of the underscore?